### ISSUE TITLE
Badge for storage kind not showing up next to compute kind in Dagster's lineage view

### ISSUE THREAD

**Issue by tobiplay** [dagster-io/dagster#24136]
### Dagster version

1.8.4

### What's the issue?

In previous versions, a badge next to the compute kind badge indicated where the data was being stored (e.g., BigQuery or S3). When using dlt, this tag was automatically inferred from dlt‘s pipeline configuration from my understanding. This badge is not being shown anymore next to the compute kind badge in the UI's lineage view, independent of what storage kind is being used.

### What did you expect to happen?

In the current version, the storage kind badge is still displayed in the assets list view and I can also filter by it in the lineage view; it's just not shown in the lineage view next to the compute badge. I’m therefore wondering if this change was intentional to clean up the UI or if it’s a bug, as the badge was still visible in the lineage UI before I upgraded my project from 1.7.16 to 1.8.0 (and above). The changelogs don't seem to mention such a change.

### Deployment type

None

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by kristianandre** [comment]
I believe they are still unsure about how to handle storage kind, so they removed it from the lineage for now. It should be an experimental feature which means it might have breaking changes without much notice.

**Comment by tobiplay** [comment]
Good call, @kristianandre. Didn't know it was still considered experimental, thanks!
